Assets/__Scripts/MapEditor/UI/Placement Controller UI/ColorTypeController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ColorTypeController : MonoBehaviour
+{
+    [SerializeField] private NotePlacement notePlacement;
+    [SerializeField] private Image leftSelected;
+    [SerializeField] private Image rightSelected;
+    [SerializeField] private Image leftNote;
+    [SerializeField] private Image leftLight;
+    [SerializeField] private Image rightNote;
+    [SerializeField] private Image rightLight;
+
+    void Start()
+    {
+        leftSelected.enabled = true;
+        rightSelected.enabled = false;
+        StartCoroutine(SetupColors());
+    }
+
+    private IEnumerator SetupColors()//PlatformDescriptor is in another scene so we have to find it, can probably change this get the colors from somewhere else in the future
+    {
+        PlatformDescriptor descriptor = null;
+        while(descriptor == null)
+        {
+            descriptor = Resources.FindObjectsOfTypeAll<PlatformDescriptor>().FirstOrDefault(x => x.isActiveAndEnabled);
+            yield return new WaitForFixedUpdate();
+        }
+        leftNote.color = descriptor.colors.RedNoteColor;
+        leftLight.color = descriptor.colors.RedColor;
```

Comment:
Another thing to note: Colors can be overridden by the user using the Chroma color panel, so this should probably be hooked up somehow.

*Another* another thing to note: Perhaps consider automatically switching the Light colors between the Boost/Non-Boost versions. I think that will be a nice touch and something people can appreciate.